project.clj

```diff
@@ -9,6 +9,8 @@
   :dependencies [[org.nanohttpd/nanohttpd "2.3.1"]]
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.8.0"]
                                   [cheshire "5.5.0"]
-                                  [clj-http-lite "0.3.0"]]}}
+                                  [clj-http-lite "0.3.0"]
+                                  ;; needed for java>=11:
+                                  [javax.xml.bind/jaxb-api "2.3.1"]]}}
```

Comment:
Had to add this to be able to run `lein test` with my java 11 setup. I can rebase the commit out if you'd rather not have it.